trillian/ctfe/storage/mysql/mysql_test.go

```diff
@@ -30,7 +30,12 @@ func TestIssuanceChainFindByKeySuccess(t *testing.T) {
 	if err != nil {
 		t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
 	}
-	defer db.Close()
+	defer func() {
+		mock.ExpectClose()
```

Comment:
You could add a WillReturnError code message in case if fails.

Reply:
I can add one more test case to expect close error in the next PR.